docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,175 @@
+
+Not all hosting scenarios permit the enforcement of rewrite rules. Additional information is provided in the hosting configuration examples that follow.
+
+## Host with ASP.NET Core
+
+ASP.NET Core apps that host Blazor apps configure the server-side app's middleware pipeline with the `UseBlazor` extension method on [IApplicationBuilder](https://docs.microsoft.com/dotnet/api/microsoft.aspnetcore.builder.iapplicationbuilder) in `Startup.Configure`:
+
+```csharp
+public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+{
+    if (env.IsDevelopment())
+    {
+        app.UseDeveloperExceptionPage();
+    }
+
+    app.UseBlazor<Client.Program>();
+}
+```
+
+The `UseBlazor` extension method performs the following tasks:
+
+* Reads the Blazor configuration file (*.blazor.config*) from the `ClientAssemblyPath`, which is either an option provided to the `UseBlazor` extension method or defaulted to the app's assembly file location.
+* Configures [Static File Middleware](https://docs.microsoft.com/aspnet/core/fundamentals/static-files) to serve Blazor's static assets from the *dist* folder. In the Development environment, the files in *wwwroot* are served.
+* Configure live reloading when in the Development environment and Debug configuration.
+* Configure single-page application routing for files outside of the *_framework* folder, which serves the default page (*index.html*) for any request that hasn't been served by a prior Static File Middleware instance.
+
+For general information on ASP.NET Core app hosting and deployment, see:
+
+* [Host and deploy ASP.NET Core](https://docs.microsoft.com/aspnet/core/host-and-deploy)
+* [Host ASP.NET Core on Azure App Service](https://docs.microsoft.com/aspnet/core/host-and-deploy/azure-apps/)
+* [.NET Core application deployment](https://docs.microsoft.com/dotnet/core/deploying/)
+* [Deploying .NET Core apps with command-line interface (CLI) tools](https://docs.microsoft.com/dotnet/core/deploying/deploy-with-cli)
+* [Deploying .NET Core apps with Visual Studio](https://docs.microsoft.com/dotnet/core/deploying/deploy-with-vs)
+
+## Docker/Linux hosting
+
+The first line of the Blazor config file (*.blazor.config*) configures the source MSBuild path to `.`. A consequence of this setting is that the webroot path is set to `.\wwwroot`, which results in published files being split between *wwwroot* and *dist* folders. For an unsupported workaround, see [On linux UseBlazor extension method fails with "The path must be absolute"](https://github.com/aspnet/Blazor/issues/376).
+
+## Static deployment strategies
+
+The following static deployment strategies make the Blazor deployment's static assets available directly for download. Conceptually, other static hosting system configurations are similar.
```

Comment:
This phrasing feels overcomplicated. Suggestion: *If you're deploying a standalone Blazor application, you can use any web server that serves static files.*

Reply:
Well, can't use "you" (unless it can't be avoided). How about ...

> When deploying a standalone Blazor app, any web server or hosting service that serves static files can host a Blazor app.

Although, we'll eat those words with Azure CDN AFAICT. I might need to take a little more time with it. I noted that `@danroth27` said in an issue comment that the `_` (underscore) prefix on the *_frameworks* folder can be removed.